source/guides/models/database-configuration.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Guides - Database Configuration
+---
+
+# Database Configuration
+
+Before starting your server, you need to configure the database link in <code>.env*</code> files.
+
+Open this file for each environment and update <code>DATABASE_URL</code> for your database.
+
+## <a href="http://www.postgresql.org/" target="_blank">PostgreSQL</a>
+
+Setup database variable for the development environment:
+
+```
+# .env.development
+DATABASE_URL="postgres://username:password@localhost/bookshelf_development"
```

Comment:
@mzelenyuk could you change `postgres://` to `postgresql://`? We are generating `postgresql://` because it supporter in MRI and JRuby ;)